qa/rolling-upgrade/src/test/java/org/opensearch/upgrades/IndexingIT.java

```diff
@@ -63,42 +69,42 @@
 
```

Comment:
A small question. Would you consider putting the test cases about mapping type removal to a separate class?
1. There is a new test case to "verify older indices are getting snapshots successfully in OpenSearch 2.0", it seems out of the range of this class "Basic test that indexed documents survive the rolling restart."
2. All the new test cases is only valid when upgrading from version < 2.0, it might be more neat to treat these test in a separate group.

Reply:
The tests are now moved into a `TypeRemovalIT` class

Reply:
Thank you for this structure change! 👍